### ISSUE TITLE
Favicon notification badge does not render correctly

### ISSUE THREAD

**Issue by federicobond** [axllent/mailpit#210]
I have some code laying around that implements this well, if you are interested I can submit a pull request for you to evaluate.

<img width="98" alt="Captura de pantalla 2023-11-23 a la(s) 13 53 54" src="https://github.com/axllent/mailpit/assets/138426/5c1ef784-5d55-4447-92bf-ac365684437f">

**Comment by axllent** [comment]
Yes thank you, I'd be very interested to see what you have @federicobond. The library I'm currently using is fairly old, but was the best one I could find at the time that didn't suffer from bad performance, and which (I thought anyway) rendered ok on all major browsers. Out of curiosity, what browser and platform are you using?

**Comment by federicobond** [comment]
That's Firefox on MacOS Sonoma 14.1. I'll look for some time these days to put something together.

**Comment by federicobond** [comment]
Thanks for the quick answer!

**Comment by axllent** [comment]
Thank you too. I realise that implementing a PR may be the most difficult thing for you, so even if you could point me in the direction of a better library that would be great (ie: I can implement it myself).

**Comment by federicobond** [comment]
Sure, I put together a quick demo here: https://github.com/federicobond/badgicon

Feel free to copy and paste the code there and adapt it to your needs, I release every right to you.

**Comment by axllent** [comment]
Thank you for this, I will definitely have a good look when I get some time (next weekend as I am away for the next 5 days). 

In the meantime, could you please explain to me what the advantage is of your library vs: Tinycon (the one Mailpit uses)? I also had a quick play in my browser and has to bump the font size from 28 to 38 to get it similar to Mailpit's one as 28 was just too small to read, so I am wondering if it is just a font selection issue (ie: you don't have Arial installed).

**Comment by axllent** [comment]
I have made a small change to the existing implementation to add a fallback font (sans-serif) to see if this helps your problem (I suspect you simply don't have Arial installed). This change has been released in v1.10.2. 

Whilst I am still open to implementing your solution, I would like to know what the advantage is of your implementation vs: the tinyicon library I'm using before I do so, and in order to use yours (or a variation of yours) I would also require your project to have a valid LICENSE file in the repo (eg: something like [this](https://github.com/axllent/mailpit/blob/develop/LICENSE)). Unfortunately just a comment in an issue stating one can use code does not legally make it open source ;-)

**Comment by parth391** [comment]
Still having the same problem

Mailpit Version: 1.10.2
Firefox Version: 120.0.1 (64-bit)
MacOS Version: 14.1.2 (23B92)

<img width="261" alt="Screenshot 2023-12-02 at 11 32 56 AM" src="https://github.com/axllent/mailpit/assets/4966579/ff97f4b6-c7d4-47c0-8f3a-ad7f502820ec">

**Comment by axllent** [comment]
@parth391 Thanks for the feedback. Can you please open [this](https://mailpit.axllent.org/test/) and confirm whether the favicon appears as it should on the same platform/browser you tested previously? Thanks.

**Comment by federicobond** [comment]
Hi, I can confirm the test link looks good in my computer too.

**Comment by parth391** [comment]
@axllent, v1.10.4 fix the issue on MacOS.

Thanks.
